map2loop/project.py

```diff
@@ -368,28 +379,7 @@ def get_sampler(self, datatype: Datatype):
         """
         return self.samplers[datatype].sampler_label
 
-    @beartype.beartype
-    def set_minimum_fault_length(self, length: float):
-        """
-        Set the cutoff length for faults to ignore
-
-        Args:
-            length (float):
-                The cutoff length
-        """
-        logger.info(f"Setting minimum fault length to {length}")    
-        self.deformation_history.set_minimum_fault_length(length)
-
-    @beartype.beartype
-    def get_minimum_fault_length(self) -> float:
-        """
-        Get the cutoff length for faults
-
-        Returns:
-            float: The cutoff length
-        """
-        return self.deformation_history.get_minimum_fault_length()
-
+    
     # Processing functions
```

Comment:
I added this back but it could be removed, it all links to the minimum_fault_length in the config file though so we only keep track of one variable